iocore/net/YamlSNIConfig.cc

```diff
@@ -92,6 +92,7 @@ TsEnumDescriptor LEVEL_DESCRIPTOR         = {{{"NONE", 0}, {"MODERATE", 1}, {"ST
 TsEnumDescriptor POLICY_DESCRIPTOR        = {{{"DISABLED", 0}, {"PERMISSIVE", 1}, {"ENFORCED", 2}}};
 TsEnumDescriptor PROPERTIES_DESCRIPTOR    = {{{"NONE", 0}, {"SIGNATURE", 0x1}, {"NAME", 0x2}, {"ALL", 0x3}}};
 TsEnumDescriptor TLS_PROTOCOLS_DESCRIPTOR = {{{"TLSv1", 0}, {"TLSv1_1", 1}, {"TLSv1_2", 2}, {"TLSv1_3", 3}}};
+TsEnumDescriptor CONTROL_DESCRIPTOR       = {{{"NONE", 0}, {"ENABLE", 1}, {"DISABLE", 2}}};
```

Comment:
What does NONE  do?

Why can't this be a boolean? Like @zwoop had in https://github.com/apache/trafficserver/pull/5719 ?

Reply:
Internally we do need an unset value so we can determine whether the h2 field in the item class was set or not.  But you're right we don't need to advertise that to the end user, so I updated the CONTROL_DESCRIPTOR and the docs to remove the NONE option.

Now whether we want http2: on/off or h2: ENABLE/DISABLE or http/2: true/false, I don't really care.